changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
+= 2.7.1 - 2024-xx-xx =
+* Update - Update readme.txt file to indicate that a new Woo Shipping plugin is available for download.
+
```

Comment:
I don't think we need a changelog for this, and no deployment is needed to roll this out.

Reply:
Removed in ab3dacaf